rust/operator-binary/src/authentication/oidc_.rs

```diff
@@ -0,0 +1,152 @@
+use std::collections::BTreeMap;
```

Comment:
This was done to avoid name conflicts with imports operator-rs but I'll switch to using aliases for the imports to avoid having these awkward file names.